### PR TITLE
PR: Add a recipe for QdarkStyleSheet

### PR DESCRIPTION
**Pull request by dalthviz**
This the QDarkStyle package a new dependency for Spyder to support a dark theme.

@ccordoba12 could you accept being a maintainer? Also, should I add someone else? maybe @dpizetta?

### CODE REVIEW

File: recipes/qdarkstyle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - qdarkstyle
+
+about:
+  home: https://github.com/ColinDuquesnoy/QDarkStyleSheet
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (ccordoba12) [inline recipes/qdarkstyle/meta.yaml]: This needs to be LICENSE.md

**Comment by ccordoba12** [comment]
I accept to be a maintainer :+1:

File: recipes/qdarkstyle/meta.yaml (hunk 1 above)
Comment (ccordoba12) [reply recipes/qdarkstyle/meta.yaml]: That's why the tests are failing.

File: recipes/qdarkstyle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (marcelotrevisani) [inline recipes/qdarkstyle/meta.yaml]: Please change this to:
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

File: recipes/qdarkstyle/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/qdarkstyle/meta.yaml]: This project is a pure python project, right? So, please add ``noarch: python`` and also add ``pip`` as a ``host`` dependency

File: recipes/qdarkstyle/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/qdarkstyle/meta.yaml]: Instead of ``build`` change that to ``host`` please

File: recipes/qdarkstyle/meta.yaml (hunk 3 above)
Comment (ccordoba12) [reply recipes/qdarkstyle/meta.yaml]: Yes, I was about to suggest that too.

File: recipes/qdarkstyle/meta.yaml (hunk 2 above)
Comment (ccordoba12) [reply recipes/qdarkstyle/meta.yaml]: What's the `-vvv` about? I haven't seen it before.

Comment (marcelotrevisani) [reply recipes/qdarkstyle/meta.yaml]: That is basically a 'full' verbose mode of ``pip``
```
-v         Give more output. Option is additive, and can be used up to 3 times.
```

**Comment by marcelotrevisani** [comment]
LGTM!
Thanks for adding this to ``conda-forge``! :tada:

**Comment by dalthviz** [comment]
Thanks for all the help @marcelotrevisani :+1:

**Comment by dpizetta** [comment]
Thank you all!

If you would like to add me too, it is ok. I'm a maintainer of QDarkStyle also, so any changes on it I could help.

**Comment by ccordoba12** [comment]
@dpizetta, now you need to add yourself with a PR as a maintainer in

https://github.com/conda-forge/qdarkstyle-feedstock

**Pull request merged**